examples/webpack/example-builder.js

```diff
@@ -13,7 +13,7 @@ const baseDir = dirname(fileURLToPath(import.meta.url));
 
 const isCssRegEx = /\.css(\?.*)?$/;
 const isJsRegEx = /\.js(\?.*)?$/;
-const importRegEx = /\s?import .*? from '([^']+)'/g;
+const importRegEx = /\s?import (?:.*? from )?'([^']+)'/g;
```

Comment:
d3-transition was not included in the package.json from `import 'd3-transition';`
This import adds to the prototype of d3-selection import, so is not used directly.